src/Algebra/Properties/Monoid/Divisibility.agda

```diff
@@ -34,6 +34,9 @@ open import Algebra.Properties.Semigroup.Divisibility semigroup public
 ∣-reflexive : _≈_ ⇒ _∣_
 ∣-reflexive = Div.∣-reflexive trans identityˡ
 
+∣∣-reflexive : _≈_ ⇒ _∣∣_
+∣∣-reflexive x≈y = ∣-reflexive x≈y , ∣-reflexive (sym x≈y)
```

Comment:
Please put this under its own section in the file, rather than in the middle of the divisibility properties.

Reply:
Done.